2022-06-23 19:58:47,765 INFO    MainThread:1591 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2022-06-23 19:58:47,769 INFO    MainThread:1591 [wandb_setup.py:_flush():76] Loading settings from /root/AI4code/wandb/settings
2022-06-23 19:58:47,769 INFO    MainThread:1591 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-23 19:58:47,769 INFO    MainThread:1591 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-06-23 19:58:47,769 INFO    MainThread:1591 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2022-06-23 19:58:47,770 INFO    MainThread:1591 [wandb_init.py:_log_setup():428] Logging user logs to /root/AI4code/wandb/run-20220623_195847-3svvjz8b/logs/debug.log
2022-06-23 19:58:47,770 INFO    MainThread:1591 [wandb_init.py:_log_setup():429] Logging internal logs to /root/AI4code/wandb/run-20220623_195847-3svvjz8b/logs/debug-internal.log
2022-06-23 19:58:47,770 INFO    MainThread:1591 [wandb_init.py:init():462] calling init triggers
2022-06-23 19:58:47,770 INFO    MainThread:1591 [wandb_init.py:init():466] wandb.init called with sweep_config: {}
config: {}
2022-06-23 19:58:47,770 INFO    MainThread:1591 [wandb_init.py:init():515] starting backend
2022-06-23 19:58:47,770 INFO    MainThread:1591 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-23 19:58:47,773 INFO    MainThread:1591 [backend.py:ensure_launched():217] starting backend process...
2022-06-23 19:58:47,805 INFO    MainThread:1591 [backend.py:ensure_launched():222] started backend process with pid: 1736
2022-06-23 19:58:47,817 INFO    MainThread:1591 [wandb_init.py:init():525] backend started and connected
2022-06-23 19:58:47,867 INFO    MainThread:1591 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2022-06-23 19:58:47,872 INFO    MainThread:1591 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-06-23 19:58:47,873 INFO    MainThread:1591 [wandb_init.py:init():596] updated telemetry
2022-06-23 19:58:48,037 INFO    MainThread:1591 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-06-23 19:58:58,890 WARNING MainThread:1591 [wandb_init.py:init():1020] interrupted
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 999, in init
    run = wi.init()
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 631, in init
    run, timeout=self.settings.init_timeout
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 223, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 324, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/conda/lib/python3.7/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/conda/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
